### PR TITLE
DOCS Update cli introduction.md - NPM package

### LINKED ISSUE #1886: CLI introduction documentation out of date
On the following page, https://docs.celo.org/v/master/command-line-interface/introduction, we should remove the old Docker instructions. I would also add full nvm instructions to help people switch to node 10:

curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash
nvm install 10
nvm use 10

On the following page, https://docs.celo.org/v/master/command-line-interface/introduction, we should tell people that they need python2 installed to install celocli.

### PR DESCRIPTION
**Pull request by KoenBal**
### Description
Updated NPM-package - Installation
Added Upgrades and Uninstall sections

### Tested
NPM Package installation was tested on Windows
Upgrades and Uninstall not tested.

### Other changes
Slight restructuring of document

### Related issues
- Fixes #1886

### CODE REVIEW

File: packages/docs/command-line-interface/introduction.md [hunk 1]
```diff
@@ -8,21 +8,73 @@ description: >-
...
 ```
+Check if the celocli is installed by requesting its version:
+```bash
+# celocli -v
+@celo/celocli/0.0.X ... node-v10.17.0
+```
 
-{% hint style="info" %}
-We are currently deploying the CLI with only Node v10.x LTS support. If you are running a different version of Node, consider using [NVM](https://github.com/nvm-sh/nvm#installation-and-update) to manage your node versions. e.g. with: `nvm install 10 && nvm use 10`
-{% endhint %}
```
Comment (koenprivate) [inline packages/docs/command-line-interface/introduction.md:21]: Information is added above.

File: packages/docs/command-line-interface/introduction.md [hunk 2]
```diff
@@ -67,5 +119,3 @@ celocli <module>:<command> <...args> <...flags?>
 The `celocli` tool assumes that users are running a node which they have access to signing transactions on. The config module is included below for convenience. Use the sidebar to navigate to the other modules.
 
 **All balances of Celo Gold or Celo Dollars returned from the CeloCLI are expressed in units of 10<sup>-18</sup>**
-
-{% embed url="https://www.npmjs.com/package/@celo/celocli" %}
```
Comment (koenprivate) [inline packages/docs/command-line-interface/introduction.md:71]: Available at the end of the installation instructions.

File: packages/docs/command-line-interface/introduction.md [hunk 3]
```diff
@@ -8,21 +8,73 @@ description: >-
 
 ## Getting Started
 
+You can install the Celo CLI as a NPM Package or if you don't have NPM or are having trouble installing the Celo CLI with your version of node, you can use a docker image that runs the Celo Blockchain client in full sync mode which includes the Celo CLI.
+
 ### NPM Package
 
-The Celo CLI is published as a node module on NPM. Assuming you have [npm installed](https://www.npmjs.com/get-npm), you can install the Celo CLI using the following command:
+#### Installation
```
Comment (koenprivate) [inline packages/docs/command-line-interface/introduction.md:15]: npm is available when installing Node using nvm as recommended below.

File: packages/docs/command-line-interface/introduction.md [hunk 4]
```diff
@@ -8,21 +8,73 @@ description: >-
...
-We are currently deploying the CLI with only Node v10.x LTS support. If you are running a different version of Node, consider using [NVM](https://github.com/nvm-sh/nvm#installation-and-update) to manage your node versions. e.g. with: `nvm install 10 && nvm use 10`
-{% endhint %}
+More info about the NPM celocli package:
+{% embed url="https://www.npmjs.com/package/@celo/celocli" %}
 
-### Docker Image
+#### Upgrades
 
-Additionally, if don't have NPM or are having trouble installing the Celo CLI with your version of node, you can use a docker image that runs the Celo Blockchain client in full sync mode which includes the Celo CLI.
+Check all the pre-requisites under installation before using the following command:
```
Comment (koenprivate) [inline packages/docs/command-line-interface/introduction.md:54]: Moved to the top of the doc.

**Comment by annakaz** [comment]
Thanks @KoenBal for your contribution to the docs! Closing this as it is out of date with the current docs, but feel free to open a new PR if the changes are still relevant

**Pull request closed without merging**